src/gpu/intel/ocl/reusable_softmax.cl

```diff
@@ -70,3 +71,158 @@ reusable_softmax_fwd_generic(__global SRC_DATA_T *src, __global DST_DATA_T *dst,
         dst[c - begin] = TO_DST(unscaled * scale);
     }
 }
+#endif
+
+#define VECT_SIZE 8
+#define DIVUP(a, b) (((a) + (b)-1) / (b))
+
+#define STORE_FLOAT8(prefix, ptr, val) \
```

Comment:
Can this be moved to ocl_types.h?

Reply:
Dropped VECT_SIZE, but the others come from gen9 softmax and sit separately. I'd rather make changes to ocl_types.h in a separate PR. You know how it is testing only softmax vs _everything_.